src/tests/ie_test_utils/common_test_utils/data_utils.hpp

```diff
@@ -194,14 +194,14 @@ void inline
 fill_data_random(T *pointer, std::size_t size, const uint32_t range = 10, int32_t start_from = 0, const int32_t k = 1,
                  const int seed = 1) {
     testing::internal::Random random(seed);
-    random.Generate(range);
+    random.Generate(range * k);
```

Comment:
Question for anyone. Just curious: what do we here? Do we skip first random value for some purpose? What is the purpose?